ipaclient/install/ipa_certupdate.py

```diff
@@ -148,12 +149,12 @@ def run_with_args(api):
             services.knownservices.krb5kdc.restart()
 
 
-def update_client(certs):
+def update_client(certs: typing.List[certstore.CACertInfo]):
     update_file(paths.IPA_CA_CRT, certs)
     update_file(paths.KDC_CA_BUNDLE_PEM, certs)
     update_file(paths.CA_BUNDLE_PEM, certs)
 
-    ipa_db = certdb.NSSDatabase(api.env.nss_dir)
+    ipa_db = certdb.NSSDatabase(paths.IPA_NSSDB_DIR)
```

Comment:
I'm not sure it was ever used but this removes the ability to override the nss directory via the environment. This backs out a change you made in https://pagure.io/freeipa/issue/6386

Reply:
I ran into a bootstrapping issue during installation. Latest version is now using `api.env.nss_dir` and fallback to `paths.IPA_NSSDB_DIR` when path is not set up.

Reply:
The problem was caused by

https://github.com/freeipa/freeipa/blob/691b5d21af823f6119a65bde27b96e01a9573d1a/ipaclient/install/client.py#L2953-L2958